docs/guide-authentication.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Authentication
+category: online projects
+order: 70
+---
+
+Online projects may require authentication to create branches or commits on the remote or even fetch the content in case of private repositories. Since abapGit uses git over HTTP the authentication mechanism is Basic Authentication (RFC 7617) using username and password. If the remote requires authentication for an action a popup will appear to enable the user to provide the necessary data. Once authentication is successful the session remains intact until abapGit is restarted.
+
```

Comment:
i think HTTPS is mandatory. add link to the "SSL setup" which is a prerequisite

Reply:
No, it's not.